### ISSUE TITLE
Making app layout responsive

### ISSUE THREAD

**Issue by heytulsiprasad** [heytulsiprasad/picturesque.css#1]
The app now works on all the platforms and browsers, however it needs to be responsive for users to get the most out of it. So, if you want to contribute something, starting with **adding responsiveness** would be a good one!

:+1: 
@heytulsiprasad

**Comment by heytulsiprasad** [comment]
**Important**: Respond to this thread before starting to work, so that others can know if anyone is already working on it.

**Comment by heytulsiprasad** [comment]
As of now, I'm closing the issue coz I think there needs to be some more features added to this project. Hopefully, anyone here can create another issue if anything comes up, or ping me on twitter, @heytulsiprasad .
